docs/ug/report-config.yaml

```diff
@@ -0,0 +1,11 @@
+repoUrl: github.com/user/repo
+reportTitle: RepoSense Report
+authorDisplayName: Sample Author
```

Comment:
Not needed

Reply:
@damithc Sure thing, will remove these from the YAML file and other associated classes in a future commit.